### PR TITLE
Use Jenkins project-wide formatting configuration

### PR DESCRIPTION
**Pull request by basil**
This repository uses a non-standard (for the Jenkins project) code formatting style and enforcement during the build. This PR switches this repository to use the Jenkins standard project-wide formatting configuration and enforcement with Spotless. The benefit is that it makes it easier for Jenkins developers to go from one project to the next without the cognitive dissonance of switching styles.

### Testing done

Ran `mvn clean verify -DskipTests`.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -373,18 +335,21 @@
...
+        <property>
+          <name>env.CI</name>
+        </property>
+      </activation>
+      <properties>
+        <!-- TODO fails on CI on Windows for inscrutable reasons -->
+        <spotless.check.skip>true</spotless.check.skip>
+      </properties>
+    </profile>
+  </profiles>
```
Comment (jtnord) [inline pom.xml]: https://github.com/jenkins-infra/helpdesk/issues/3865

Comment (basil) [reply pom.xml]: This message does not state the reason why this PR was not approved, or the action that needed to be taken in order for this PR to be approved.

Comment (basil) [reply pom.xml]: commit ce65d9f

**Comment by basil** [comment]
@jtnord Is this PR ready for merge?

**Review by jtnord — APPROVED** [review]

**Comment by jtnord** [comment]
> @jtnord Is this PR ready for merge?

thank you.  I take it the windows spotless issue is somehow constrained to the VM images then?

**Comment by basil** [comment]
> I take it the windows spotless issue is somehow constrained to the VM images then?

Seems that way? I haven't looked into it very closely.

**Pull request merged**